website/docs/integrations/flink/flink1.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 4
+title: Flink 1.x
+---
+
```

Comment:
Can we add warning here that the doc here is for Flink 1? I know it will be in the sidebar but would not hurt us to have this.